src/styles/components/_progress.scss

```diff
@@ -0,0 +1,45 @@
+$LINEAR_ANIMATION_TIME: 2.5s;
+
+@keyframes hold-visibility {
+  from { visibility: visible; }
+  to   { visibility: visible; }
+}
+
+@keyframes loading-progress-before {
+  from { transform: translate(-50%); }
+  to   { transform: translate(100vw) translate(-50%); }
+}
+
+.w-loading-progress {
+  background: #3fc4ff;
+  height: 2px;
+  left: 0;
+  pointer-events: none;
+  position: fixed;
+  right: 0;
+  top: 0;
+  transform: scaleY(1);
+  transform-origin: 50% 0%;
+  transition: transform $TRANSITION_SPEED ease;
+  transition-delay: 1s;  // show loading hint only after waiting this long
+  z-index: 100000;
+
+  // Shows a white "flurry" on top of the loading indicator.
```

Comment:
I think there may not be enough contrast between the foreground and background in the animation. I was actually staring at it for a sec and came back to look at the CSS because I remembered the "flurry" comment but didn't think I was seeing it.

Reply:
I've made the flurry go to full-white in the middle. I think it's more obvious now. You need to really delay the load speed to see this ever show up.